db/schema.rb

```diff
@@ -33,6 +33,11 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "pgmigrations", id: :integer, default: nil, force: :cascade do |t|
```

Comment:
I think this is coming from an old version of the db. This was a migration table for the postgres migration node library I used briefly. Do you happen to have this table in your local dev db? If so, delete it and rerun `rails db:migrate`. That should remove this line.

Reply:
I don't know why it's inserting that. Dropping the pgmigrations tables doesn't seem to address the problem?

Reply:
I'll destroy the db and try again